src/api/versions/v1/services/scores-service.ts

```
@@ -50,6 +50,13 @@ export class ScoresService {
 
     const { score } = request;
 
+    // Sum old score if it exists
+    const oldScoreKV = await this.kvService.getScore(userName);
+
+    if (oldScoreKV) {
+      score += oldScoreKV.score;
```

Biome (1.9.4)

[error] Can't assign score because it's a constant

This is where the variable is defined as constant

Unsafe fix: Replace const with let if you assign it to a new value.


(lint/correctness/noConstAssign)